var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -12,30 +36,97 @@ class Adios(Package):
     """
 
     homepage = "http://www.olcf.ornl.gov/center-projects/adios/"
-    url = "https://github.com/ornladios/ADIOS/archive/v1.9.0.tar.gz"
+    url      = "https://github.com/ornladios/ADIOS/archive/v1.10.0.tar.gz"
+
+    # make -j is currently broken
+    parallel = False
+
+    version('1.10.0', 'eff450a4c0130479417cfd63186957f3')
+    version('1.9.0' , '310ff02388bbaa2b1c1710ee970b5678')
+
+    variant('shared', default=True,
+            description='Builds a shared version of the library')
 
-    version('1.9.0', '310ff02388bbaa2b1c1710ee970b5678')
+    variant('fortran', default=True,
+            description='Enable Fortran bindings support')
+
+    variant('mpi', default=False, description='Enable MPI support')
+    variant('infiniband', default=False, description='Enable infiniband support')
```

Comment:
No it's not necessary. On Titan, we need the opposite --without-infiniband
to avoid using it and use the Cray Gemini/Aries networking when building
ADIOS wih staging methods.
File based methods don't use network libs, only MPI.

On Monday, August 29, 2016, Axel Huebl <notifications@github.com> wrote:

> In var/spack/repos/builtin/packages/adios/package.py
> <https://github.com/LLNL/spack/pull/1652#discussion_r76690738>:
>
> >
> > -    version('1.9.0', '310ff02388bbaa2b1c1710ee970b5678')
> > +    variant('fortran', default=True,
> > +            description='Enable Fortran bindings support')
> > +
> > +    variant('mpi', default=False, description='Enable MPI support')
> > +    variant('infiniband', default=False, description='Enable infiniband support')
>
> @pnorbert <https://github.com/pnorbert> is that --with-infiniband option
> necessary to be set manually? I just copied it from the previous code but
> never used it before (gosh, maybe I should have on our local cluster...?)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/1652/files/b33777fd2f7eb377727cead7f38ccdf63cd3a71d#r76690738>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADGMLWXqBYnOfzhUAWYwWxgJOYTBkbaLks5qk06ogaJpZM4Jvkjc>
> .
>